owncloud/ios-app
# [feature/filelist-layout] Cleaner File List Layout

## Description
Changed file list cell layout to get more space for the filename and more aligned icons, less jumpy

## Motivation and Context
As you can see on the screenthos below, there are more space for file names and the icons for state shared, link is no longer jumpy behind the file name.

| Before Changes                                                                                                                                                        | After Changes                                                                                                                                                         |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-01-13 at 12 07 07](https://user-images.githubusercontent.com/736109/72251270-d97bf900-35fd-11ea-873b-8c0550e7f02a.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-01-13 at 13 50 40](https://user-images.githubusercontent.com/736109/72257343-ba846380-360b-11ea-8709-ed97917665ef.png) |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-01-13 at 12 06 59](https://user-images.githubusercontent.com/736109/72257145-3500b380-360b-11ea-9abb-aa3caddd3313.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-01-13 at 13 52 18](https://user-images.githubusercontent.com/736109/72257438-f4ee0080-360b-11ea-9822-49786089f6df.png) |

## How Has This Been Tested?
- Open file list
- check layout of the icons
- change state of shared, public link

## Screenshots (if appropriate):

## Types of changes


## Checklist:



____

QA

Test Plan:

https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%201.3.0/New%20List%20Layout.md

BUGS & IMPROVEMENTS:

- [ ] (1) Marks at the right https://github.com/owncloud/ios-app/pull/594#issuecomment-576247312
